android/src/main/AndroidManifest.xml

```diff
@@ -16,5 +16,19 @@ limitations under the License.
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="com.squareup.sdk.reader.react">
+          
+    <uses-permission
+        android:name="android.permission.BLUETOOTH"
```

Comment:
Why do we need permission for this file?

Reply:
For support above android 12 versions.